recipes/re2/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "re2" %}
+{% set version = "2016.11.01" %}
+{% set dashversion = version.replace('.', '-') %}
+{% set sha256 = "01ee949f03e1c4057dc533cf139f967fb1b427015769d53b9ee07757631e9669" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://github.com/google/{{ name }}/archive/{{ dashversion }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - win-cmakelists.patch  # [win]
+
+build:
+  number: 0
+  skip: True  # [not py35]
```

Comment:
Apparently missed this. 😔

It is causing all builds to be skipped in the feedstock. Please remove this `skip` and re-render in a fresh PR from your fork.